### PR TITLE
Closes #1464 - add examples to New-ItemProperty

### LINKED ISSUE #1464: Help for New-ItemProperty needs additional examples
Issue Details
-----------------
The acceptable values for the PropertyType parameter are not enumerated by intellisense.  Those values are discoverable in the parameters section of the help.  But since the values (at least for the registry provider) are not quite you might expect if you are used to looking at Regedit, I think it would be helpful to have at least one example of creating a new registry value that is of an atypical type, such as "MultiString"

`New-ItemProperty -Path "HKLM:\SOFTWARE\Mine\SomeApp\" -Name 'Server' -PropertyType MultiString -Value ('a','b','c')`

Version(s) of document impacted
------------------------------
- [X] Impacts 6 document
- [X] Impacts 5.1 document
- [X] Impacts 5.0 document
- [X] Impacts 4.0 document
- [X] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------
- [ ] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

File: reference/3.0/Microsoft.PowerShell.Management/New-ItemProperty.md [hunk 1]
```diff
@@ -53,31 +61,65 @@ NoOfEmployees : 822
...
+
+The first part of the command uses the `Get-Item` cmdlet to get the MyCompany registry key.
+The pipeline operator sends the results of the command to `New-ItemProperty`, which adds the new registry entry, NoOfLocations, and its value, 3, to the MyCompany key.
+
+This command works because the parameter-binding feature of Windows PowerShell associates the path of the `RegistryKey` object that `Get-Item` returns with the *LiteralPath* parameter of `New-ItemProperty`.
+For more information, see [about_Pipelines](../Microsoft.PowerShell.Core/About/about_pipelines.md).
+
+### Example 3: Create a MultiString value in the registry
+
+```powershell
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Management/New-ItemProperty.md]: This is mixed code and output. So it is not copy/paste runnable.

File: reference/3.0/Microsoft.PowerShell.Management/New-ItemProperty.md [hunk 2]
```diff
@@ -53,31 +61,65 @@ NoOfEmployees : 822
...
+$newValue = New-ItemProperty -Path "HKLM:\SOFTWARE\ContosoCompany\" -Name 'MultiString' -PropertyType MultiString -Value ('a','b','c')
+$newValue.multistring[0]
+```
+
+```output
+a
+```
+
+These two examples show two ways to create `MultiString` values with `New-ItemProperty`.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Management/New-ItemProperty.md]: These three sentences should be one paragraph. The alternative is to split Example 3 into two examples. One using Here-String and Example 4 using an array. I think I prefer splitting this into two examples. Which every you choose, make the same change in all versions.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/New-ItemProperty.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1464/reference/3.0/Microsoft.PowerShell.Management/New-ItemProperty.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/new-itemproperty?view=powershell-3.0&branch=pr-en-us-2627) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/New-ItemProperty.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1464/reference/4.0/Microsoft.PowerShell.Management/New-ItemProperty.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/new-itemproperty?view=powershell-4.0&branch=pr-en-us-2627) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/New-ItemProperty.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1464/reference/5.0/Microsoft.PowerShell.Management/New-ItemProperty.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/new-itemproperty?view=powershell-5.0&branch=pr-en-us-2627) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/New-ItemProperty.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1464/reference/5.1/Microsoft.PowerShell.Management/New-ItemProperty.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/new-itemproperty?view=powershell-5.1&branch=pr-en-us-2627) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/New-ItemProperty.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1464/reference/6/Microsoft.PowerShell.Management/New-ItemProperty.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/new-itemproperty?view=powershell-6&branch=pr-en-us-2627) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C28%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806281631169247-2627%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vSeOQiX2LqM2r1hp1UUAcK1xoy%2FMewcQbJUpczw9NvM%3D&st=2018-06-28T16%3A36%3A27Z&se=2018-07-29T16%3A41%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Management/New-ItemProperty.md [hunk 3]
```diff
@@ -53,31 +61,65 @@ NoOfEmployees : 822
...
+$newValue = New-ItemProperty -Path "HKLM:\SOFTWARE\ContosoCompany\" -Name 'HereString' -PropertyType MultiString -Value @"
+This is text which contains newlines
+It can also contain "quoted" strings
+"@
+$newValue.multistring
+```
+
+This example creates a MultiString value using a Here-String.
+
+### Example 4: Create a MultiString value in the registry using an array
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Management/New-ItemProperty.md]: You split the examples on the wrong line. The output of the previous example is in the code block of this example.

Comment (bobbytreed) [reply reference/3.0/Microsoft.PowerShell.Management/New-ItemProperty.md]: fixed

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/New-ItemProperty.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1464/reference/3.0/Microsoft.PowerShell.Management/New-ItemProperty.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/new-itemproperty?view=powershell-3.0&branch=pr-en-us-2627) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/New-ItemProperty.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1464/reference/4.0/Microsoft.PowerShell.Management/New-ItemProperty.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/new-itemproperty?view=powershell-4.0&branch=pr-en-us-2627) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/New-ItemProperty.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1464/reference/5.0/Microsoft.PowerShell.Management/New-ItemProperty.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/new-itemproperty?view=powershell-5.0&branch=pr-en-us-2627) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/New-ItemProperty.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1464/reference/5.1/Microsoft.PowerShell.Management/New-ItemProperty.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/new-itemproperty?view=powershell-5.1&branch=pr-en-us-2627) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/New-ItemProperty.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1464/reference/6/Microsoft.PowerShell.Management/New-ItemProperty.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/new-itemproperty?view=powershell-6&branch=pr-en-us-2627) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C28%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806281655133414-2627%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=bYOiK1tMp59Il3%2BayDKgPMwrQJfCszf0EF8KT%2FxbJ4U%3D&st=2018-06-28T17%3A00%3A34Z&se=2018-07-29T17%3A05%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**